webapp/update-shared-packages-npm.sh

```
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+VERSION=${1:-11.7.0-0}
+
+npm add @mattermost/types@${VERSION} @mattermost/client@${VERSION} mattermost-redux@${VERSION} @mattermost/shared@${VERSION}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)